packages/api/src/logger/index.ts

```diff
@@ -0,0 +1,194 @@
+import pino, {
+  BaseLogger,
+  DestinationStream,
+  LevelWithSilent,
+  LoggerOptions,
+  redactOptions,
+} from 'pino'
+
+/**
+ * Determines if log environment is development or test
+ */
+const isDevelopment =
+  process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test'
+
+/**
+ * Determines if log environment is production by checking if not development
+ */
+const isProduction = !isDevelopment
```

Comment:
I think it's safer to check if `process.env.NODE_ENV === "production", since the default is that NODE_ENV is undefined. (Although we do set it explicitly via `yarn rw dev`.)

Reply:
I don't agree. If it's `undefined` I think it's safer to default to `isProduction = true`

Reply:
Wait ...

> we do set it explicitly via yarn rw dev`.

```
api | # logger configuration #
api | isProduction: false
api | isDevelopment: true
api | isPretty: true
api | isFile: false
api | isStream: false
api | logLevel: trace
```

Is what I get currently when I run `yarn rw dev` which  is what I expect.

My `process.env.NODE_ENV` is:

```
api | development
api | false
```

Is that unexpected?

Reply:
@dthyresson It was broken. I fixed it this morning: https://github.com/redwoodjs/redwood/blob/main/packages/cli/src/commands/dev.js#L81

Reply:
> I don't agree. If it's undefined I think it's safer to default to isProduction = true

@Tobbe You're probably right, I was just talking about the logic here, but not the implication of the change.